src/components/TodoItem.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { Dispatch, FC } from 'react';
+
+import { Todo } from '../types/Todo';
+import classNames from 'classnames';
+import { deleteTodos } from '../api/todos';
+
+interface Props {
+  todo: Todo;
+  onErrorMessage: (message: string) => void;
+  setTodos: Dispatch<React.SetStateAction<Todo[]>>;
+  setDeletingId: Dispatch<React.SetStateAction<number>>;
+  deletingId: number;
+}
+const TodoItem: FC<Props> = ({
+  todo,
+  onErrorMessage,
+  setTodos,
+  setDeletingId,
+  deletingId,
+}) => {
+  const handleDeleteTodo = (id: number) => {
+    setDeletingId(id);
+    deleteTodos(id)
+      .then(() =>
+        setTodos((prevState: Todo[]) => prevState.filter(t => t.id !== id)),
```

Comment:
One letter in a callback is not a good idea. Please, fix it in the feature iteration.